src/views/Transactions.vue

```diff
@@ -40,7 +40,10 @@ export default {
     ...mapState({
       total: state => state.wallet.balance,
       txLabels: state => state.wallet.txLabels,
-      transactions: state => state.wallet.transactions,
+      transactions: state =>
+        state.wallet.transactions.sort(
+          (t1, t2) => new Date(t2.timestamp) - new Date(t1.timestamp),
```

Comment:
Timestamps are integers, you can subtract them:

```suggestion
          (t1, t2) => t2.timestamp - t1.timestamp,
```